net-analyzer/gvm-libs/gvm-libs-22.6.3-r1.ebuild

```diff
@@ -53,6 +53,8 @@ BDEPEND="
 
 src_prepare() {
 	cmake_src_prepare
+	# QA-Fix | Remove -Werror compiler flag
+	sed -i -e "s/-Werror//" "${S}"/CMakeLists.txt || die #909558
```

Comment:
This does not need a revision bump. See https://devmanual.gentoo.org/general-concepts/ebuild-revisions/index.html. I'd also move the bug number in the comment line above the `sed`. Right now, it could be mistaken for an argument to `die`.